Unity/Champloo/Assets/Scripts/Player/Movement States/OnWall.cs

```diff
@@ -4,51 +4,60 @@
 public class OnWall : MovementState
 {
     [SerializeField] private float maxFallSpeed = 10f;
-    [SerializeField]
-    private Vector2 wallJumpClimbForces;
-    [SerializeField]
-    private Vector2 wallJumpOffForces;
-    [SerializeField]
-    private Vector2 wallLeapForces;
-    [SerializeField]
     private float wallStickTime;
 
     [SerializeField]
+    private Vector2 wallJumpVelocity;
```

Comment:
Overall I see why we're doing the configuration like this, but I'm not sure this is going to be enough to let us configure it as we need. We might want to pull out the modifiers for each scenario into accessible variables.